examples/cbyte-nocase.peg

```diff
@@ -31,6 +31,6 @@ RepOp       {L} <- [*+?]
 Dot         {L} <- '.'
 NoCase      {L} <- 'i' !Identifier
 EndLine     {L} <- '\r\n' / '\n\r' / '\n' / '\r'
-Comment     {L} <- '#' (!EndLine .)* EndLine
+Comment     {L} <- '#' (!EndLine .)* EndLine?
```

Comment:
Fixed in https://github.com/ChrisHixon/chpeg/commit/248b1b1802f273000a17a71874ca5e2cf20ae02a